packages/protocol/contracts/L1/libs/LibProposing.sol

```diff
@@ -19,6 +19,13 @@ library LibProposing {
     bytes32 private constant _EMPTY_ETH_DEPOSIT_HASH =
         0x569e75fc77c1a856f6daaf9e69d8a9566ca34aa47f9133711ce065a571af0cfd;
 
+    /// @notice bytes32 representation of the string "tier_provider".
+    bytes32 private constant TIER_PROVIDER = bytes32("tier_provider");
+    /// @notice bytes32 representation of the string "proposer".
+    bytes32 private constant PROPOSER = bytes32("proposer");
+    /// @notice bytes32 representation of the string "taiko_token".
+    bytes32 private constant TKO = bytes32("taiko_token");
```

Comment:
should we enforce some name convention here for **bytes32 strings**:

- name them STR_SOME_THING = "some Thing" if it is public or internal, same for "SOME THING"
- name them _STR_SOME_THING if it is private.

Reply:
OK, tho in practice they will be public (so that other Libraries could see) or if libraries inheriting from libraries it could be private, but i dont think inheritance is necessary.

Reply:
I think internal should be good enough as these library will all be inlined to TaikoL1.sol now upon deployment. There is no public/external methods now.